File: python/q8.py

arr = [5, 3, 8, 4, 2] 

n = len(arr)  

for i in range(n):     
    for j in range(n):  
        if arr[j] > arr[j + 1]:             
            temp = arr[j]             
            arr[j] = arr[j + 1]             
            arr[j + 1] = temp  
print(arr)

File: JS/q8.js
let arr = [5, 3, 8, 4, 2]; 
let n = arr.length;  
for (let i = 0; i < n; i++) 
{     
	for (let j = 0; j < n; j++) 
	{         
		if (arr[j] > arr[j + 1]) 
		{               
			let temp = arr[j];             
			arr[j] = arr[j + 1];             
			arr[j + 1] = temp;         
		}     
	   }   
}  
console.log(arr);